app/services/openid_connect_redirector.rb

```diff
@@ -60,7 +60,7 @@ def validate_redirect_uri
   end
 
   def validate_redirect_uri_matches_sp_redirect_uri
-    return if redirect_uri_matches_sp_redirect_uri?
+    return true if redirect_uri_matches_sp_redirect_uri?
```

Comment:
we can remove this if we use `.valid?` instead of `validate` -- see above (we're not used to using the return values from validator methods, and that way we don't have to worry about the ordering of them either)